src/App.css

```diff
@@ -23,6 +23,13 @@
 }
 
 @keyframes App-logo-spin {
```

Comment:
please use styled components for the keyframes. Delete all the css and use the styled components